src/components/reports/ReportForm.test.tsx

```diff
@@ -1,13 +1,18 @@
 import ReportForm from "./ReportForm";
 import {render, screen, within} from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
+import {Patient} from "@smile-cdr/fhirts/dist/FHIR-R4/classes/patient";
```

Comment:
I guess this `smile-cdr` library exposes some data types to conform to FHIR standards?

Reply:
Yep that's right. They have implemented the data models to each version of the standard either as a class or interface.